docs/CHANGELOG.md

```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+Added support for latest version of libocr with the V2 networking stack. New env vars to configure this are:
+
+```
```

Comment:
Are all of these required or are some optional?

Reply:
All are optional. Added a note about this.